### ISSUE TITLE
spelling mistake

### ISSUE THREAD

**Issue by s-monisha** [GreatDevelopers/LibreHatti#442]
>Reports>Saved registerss

**Comment by jasvir99** [comment]
Hi s-monisha. I appreciate you efforts of opening issues for LH. You can open single issue for all spelling mistakes. If you find more spelling mistakes, you can edit issue and add more into it.

As far as these spelling mistakes are concerned, you can specify verbose_name_plural in meta class of model class.
https://github.com/GreatDevelopers/LibreHatti/blob/master/src/librehatti/catalog/models.py#L39

I'll wait for pull request. Don't forget to specify issue number in commit message.

**Comment by s-monisha** [comment]
On Mon, Aug 31, 2015 at 11:25 PM, jasvir99 <notifications@github.com> wrote:

> Hi s-monisha. I appreciate you efforts of opening issues for LH. You can
> open single issue for all spelling mistakes. If you find more spelling
> mistakes, you can edit issue and add more into it.
>
>

 I thought it will be one or two mistakes so i create new issue for each
but it has may so now i'm putting all mistakes in one.
<https://github.com/GreatDevelopers/LibreHatti/issues/442#issuecomment-136444314>
Thanku.



-- 
--
Monisha
Blog:https://manishasandal786.wordpress.com/

**Comment by jasvir99** [comment]
No problem.

**Comment by manpreet9112** [comment]
verbose_name_plural is already specified in meta class?. Can I send PR from master to librehatti/master for minor mistakes?.

**Comment by jasvir99** [comment]
No. It'll lead to merge conflicts between master and dirty branch.

**Comment by manpreet9112** [comment]
Ok
